packages/frontend-core/src/components/grid/stores/rows.js

```diff
@@ -246,6 +246,14 @@ export const deriveStores = context => {
       }
     } else {
       // Some other error - just update the current cell
+      if (get(focusedCellId)) {
+        validation.actions.setError(
+          get(focusedCellId),
+          error?.message || "Error"
+        )
+      } else {
+        get(notifications).error(error?.message || "An unknown error occurred")
+      }
```

Comment:
We don't actually need to check here at all for a focused cell ID - when you press submit it will blur the focused cell, so we'll never have one unless they use they keyboard. I think we just update this to always show a notification 👍 We can also remove the line after this, which I think you just left in by mistake.

Reply:
Sorted that!